editor/plugins/particles_2d_editor_plugin.cpp

```diff
@@ -90,6 +95,13 @@ void Particles2DEditorPlugin::_generate_visibility_rect() {
 
 	EditorProgress ep("gen_aabb", TTR("Generating AABB"), int(time));
 
+	bool was_emitting = particles->is_emitting();
+	if (!was_emitting) {
+		particles->set_emitting(true);
+		OS::get_singleton()->delay_usec(1000);
+	}
+	running = 0.0;
```

Comment:
`running` is already 0, it's defined above and not modified. This was also a mistake in #20977, could you remove both?

Reply:
@akien-mga removed